sdk/azcore/log_test.go

```diff
@@ -9,21 +9,23 @@ import (
 	"fmt"
 	"net/http"
 	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/logger"
 )
 
 func TestLoggingDefault(t *testing.T) {
 	// ensure logging with nil listener doesn't fail
-	Log().SetListener(nil)
-	Log().Write(LogRequest, "this should work just fine")
+	SetListener(nil)
+	logger.Log().Write(logger.LogRequest, "this should work just fine")
```

Comment:
Since this test is in azcore, should you be testing with `azcore.LogRequest`? Same for the other classifications in these tests.

Reply:
This calls into the logger module so it has to use the type definitions located in that module.